docs/INSTALL.md

```diff
@@ -50,15 +52,30 @@ Then:
 
 **At this point you should see `(jmvenv)` at the beginning of your command prompt.**
 
+Then build and install a local copy of libsecp256k1 for python-bitcointx:
+
+    mkdir -p deps
+    cd deps
+    git clone git://github.com/bitcoin-core/secp256k1
+    cd secp256k1
+    git checkout 0d9540b13ffcd7cd44cc361b8744b93d88aa76ba
+    make clean
+    ./autogen.sh
+    ./configure --prefix JM_ROOT --enable-module-recovery --disable-jni --enable-experimental --enable-module-ecdh --enable-benchmark=no
+    make
+    make check
+    make install
```

Comment:
This leaves us inside `deps/secp256k1`, but the next instruction (`pip install -r requirements/base.txt`) cannot be launched from here. I think we should add `cd ../..` (or something) after `make install`, exactly like happens with libsodium instructions.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/781/commits/60b31e276bb4929855e6bcc4b4a27b076c1386f1